compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -896,6 +898,35 @@ void QuantizeWithMinMaxPass::set_output_type(loco::Graph *g) const
   }
 }
 
+LayerInfoMap QuantizeWithMinMaxPass::create_layer_info_map(loco::Graph *g) const
+{
+  auto info_by_name = LayerInfoMap();
+
+  for (auto &&info : _ctx->layers_info)
+  {
+    auto name = info.name;
+    bool found = false;
+    for (auto node : loco::active_nodes(loco::output_nodes(g)))
+    {
+      auto cnode = loco::must_cast<luci::CircleNode *>(node);
+      if (cnode->name() == name)
+      {
+        info_by_name[name] = &info;
+        found = true;
+        break;
+      }
+    }
+
+    if (not found)
+      throw std::runtime_error("No such layer named " + name +
+                               ". Check layer names in the quantization configuration file.");
+  }
+
+  assert(info_by_name.size() == _ctx->layers_info.size()); // FIX_ME_UNLESS
```

Comment:
Current logic itself has no problem :)
But if we apply following codes, we may be able to expect :D
- Better readability
- Better performance : O(n^2) --> O(n)

```suggestion
  for (auto &&info : _ctx->layers_info)
    info_by_name[info.name] = &info;

  for (auto node : loco::active_nodes(loco::output_nodes(g)))
  {
    auto cnode = loco::must_cast<luci::CircleNode *>(node);
    if (info_by_name.find(cnode->name()) == info_by_name.end())
    {
      throw std::runtime_error("No such layer named " + cnode->name() +
                               ". Check layer names in the quantization configuration file.");
    }
  }
```

Reply:
Looks much better!

Reply:
Hmm.. this logic causes an error. It throws an exception whenever a graph has a node not in `info_by_name`, so I roll backed the code.